lib/pages/home_page.dart

```diff
@@ -24,13 +26,35 @@ class _HomePageState extends State<HomePage> {
     });
   }
 
+  void _goToCalendarAttendancePage() async {
+    setState(() {
+      _isTransitioning = true;
+    });
+    await Future.delayed(Duration(milliseconds: 800));
```

Comment:
Make it faster

Reply:
add `const` when necessary:

   const Duration(milliseconds: 800));